src/carriers/zfp_portmonitor/CMakeLists.txt

```diff
@@ -8,24 +8,34 @@ yarp_prepare_plugin(zfp TYPE ZfpMonitorObject
                         DEPENDS "CREATE_OPTIONAL_CARRIERS;ENABLE_yarpcar_portmonitor")
 
 if(NOT SKIP_zfp)
-  set(CMAKE_INCLUDE_CURRENT_DIR ON)
-  get_property(YARP_OS_INCLUDE_DIRS TARGET YARP_OS PROPERTY INCLUDE_DIRS)
-  get_property(YARP_sig_INCLUDE_DIRS TARGET YARP_sig PROPERTY INCLUDE_DIRS)
-  include_directories(${YARP_OS_INCLUDE_DIRS} ${YARP_sig_INCLUDE_DIRS})
-  include_directories(SYSTEM ${ZFP_INCLUDE_DIRS})
-
-  yarp_add_plugin(yarp_pm_zfp zfpPortmonitor.cpp
-                              zfpPortmonitor.h)
-  target_link_libraries(yarp_pm_zfp YARP_OS YARP_sig ${ZFP_LIBRARIES})
-
-  yarp_install(TARGETS yarp_pm_zfp
-               EXPORT YARP
-               COMPONENT runtime
-               LIBRARY DESTINATION ${YARP_DYNAMIC_PLUGINS_INSTALL_DIR}
-               ARCHIVE DESTINATION ${YARP_STATIC_PLUGINS_INSTALL_DIR})
-  yarp_install(FILES zfp.ini
-               COMPONENT runtime
-               DESTINATION ${YARP_PLUGIN_MANIFESTS_INSTALL_DIR})
-
-  set_property(TARGET yarp_pm_zfp PROPERTY FOLDER "Plugins/Port Monitor")
+
+    if(ZFP_FOUND)
```

Comment:
If you set `YarpPlugin_VERBOSE` the output is more explicit (but I think there isn't an option for enabling that from the GUI)

Anyway this is disappearing, the output you want to give is mostly the same produced by FeatureSummary, and I already have a branch that does it for all the branches and will be merged before the release, so please revert this change as well to save me from looking at this again when I merge the patch.